lib/SystemStatistics.php

```diff
@@ -99,4 +110,41 @@ protected function getMemoryUsage() {
 		];
 	}
 
+	/**
+	 * Get some info about installed apps, including available updates.
+	 *
+	 * @return array data about apps
+	 */
+	protected function getAppsInfo() {
+
+		// sekeleton about the data we return back
+		$info = [
+			'num_installed' => 0,
+			'num_enabled' => 0,
+			'num_updates_available' => 0,
+			'app_updates' => [],
+		];
+
+		// load all apps
+		$apps = (new OC_App())->listAllApps();
```

Comment:
Could you use `\OCP\IAppManager::getInstalledApps()` instead? (https://github.com/nextcloud/server/blob/591e75df5c3acf51e6968f20b1856481ee56f4de/lib/public/App/IAppManager.php#L116-L116) It's a list of app ids. And then also the CI job succeeds, because you don't use private namespace anymore ;)